src/components/tree/tree.e2e.ts

```diff
@@ -904,5 +904,50 @@ describe("calcite-tree", () => {
 
       expect(await getActiveElementId(page)).toEqual("child-1");
     });
+
+    it("renders when tree is in the root node (there is no parent)", async () => {
+      const templateHTML = html` <template id="test-tree-element-template">
+        <calcite-tree>
+          <calcite-tree-item>Child 1</calcite-tree-item>
+          <calcite-tree-item>
+            Child 2
+            <calcite-tree slot="children">
+              <calcite-tree-item>Grandchild 1</calcite-tree-item>
+              <calcite-tree slot="children">
+                Grandchild 2
+                <calcite-tree-item>Great-grandchild 1</calcite-tree-item>
+                <calcite-tree-item>Great-grandchild 2</calcite-tree-item>
+              </calcite-tree>
+            </calcite-tree>
+          </calcite-tree-item>
+          <calcite-tree-item id="three">3</calcite-tree-item>
+        </calcite-tree>
+      </template>`;
+
+      const page = await newE2EPage();
+      await page.setContent(html`${templateHTML} <test-tree-element></test-tree-element>`);
```

Comment:
Nitpick: for readability, can you move the template and test element to separate lines?

```ts
html`
  ${templateHTML}
  <test-tree-element></test-tree-element>
`
```
?